### PR TITLE
add volume mode field to constructed volume spec for CSI plugin

### PR DESCRIPTION
**Pull request by wenlxie**
Add volume mode filed to constructed Volume Spec for CSI plugin

```release-note
Add volume mode filed to constructed volume spec for CSI plugin
```

### CODE REVIEW

File: pkg/volume/csi/csi_plugin.go [hunk 1]
```diff
@@ -248,6 +248,7 @@ func (p *csiPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.S
 
 	glog.V(4).Info(log("plugin.ConstructVolumeSpec extracted [%#v]", volData))
 
+	fsMode := api.PersistentVolumeFilesystem
```
Comment (msau42) [inline pkg/volume/csi/csi_plugin.go]: Add a unit test

**Comment by msau42** [comment]
I think release note is not needed.

File: pkg/volume/csi/csi_plugin.go (hunk 1 above)
Comment (wenlxie) [reply pkg/volume/csi/csi_plugin.go]: @msau42 Added

**Comment by wenlxie** [comment]
@kubernetes/sig-storage-pr-reviews

**Comment by wenlxie** [comment]
@kubernetes/sig-storage-pr-reviews

**Pull request merged**